src/ast/mod.rs

```diff
@@ -9211,6 +9253,41 @@ pub enum CopyIntoSnowflakeKind {
     Location,
 }
 
+/// Return (MsSql)
+///
+/// for Functions:
+/// RETURN scalar_expression
+///
+/// See <https://learn.microsoft.com/en-us/sql/t-sql/statements/create-function-transact-sql>
+///
+/// for Triggers:
+/// RETURN
+///
+/// See <https://learn.microsoft.com/en-us/sql/t-sql/statements/create-trigger-transact-sql>
```

Comment:
```suggestion
/// Represents a `Return` statement.
///
/// [MsSql triggers](https://learn.microsoft.com/en-us/sql/t-sql/statements/create-trigger-transact-sql)
/// [MySql functions](https://learn.microsoft.com/en-us/sql/t-sql/statements/create-function-transact-sql)
```
this makes it possible to include docs to other dialects like bigquery/snowflake that support this statement

Reply:
Done 👍 

---

BTW, see how easy it is for "MsSql" to accidentally become "M**y**Sql"? Another vote for renaming the dialect 😆 

<img width="303" alt="image" src="https://github.com/user-attachments/assets/c99b0c0f-e2ac-4e8d-85be-08861b376610" />